x-pack/legacy/plugins/monitoring/public/views/logstash/node/pipelines/index.js

```diff
@@ -24,7 +24,7 @@ import { PipelineListing } from '../../../../components/logstash/pipeline_listin
 import { DetailStatus } from '../../../../components/logstash/detail_status';
 import { CODE_PATH_LOGSTASH } from '../../../../../common/constants';
 
-const getPageData = ($injector) => {
+const getPageData = ($injector, _api = undefined, routeOptions = {}) => {
```

Comment:
I don't understand `_api = undefined` here. It doesn't appear that it's used?

Reply:
It's still used in the base classes to skip `api` eg: https://github.com/elastic/kibana/pull/46587/files#diff-cdbb606ca880715e6c1bd693d445dd62R123. Sometimes you need default values to skip optionals/unused with a `_`/`undefined`, since javascript does not have overloads